1ere passe de travail sur NeTEx Nouveaux modes

Henault opened the pull request.

Commit:
Création du fichier

Commit:
Création du fichier

Commit:
Modification de test

Commit:
Modification de test

Commit:
1er cadre

Reprise de la structure des autres profils NeTEx FR

Commit:
1er cadre

Reprise de la structure des autres profils NeTEx FR

Commit:
Update index.md

Commit:
Update index.md

Commit:
Squelette du document

Proposition de structure et essai de redaction pour voir le rendu

Commit:
Squelette du document

Proposition de structure et essai de redaction pour voir le rendu

thbar committed:
Merge branch 'main' into netex-nouveaux-modes

thbar commented:
J'ai mis à jour la branche de `master` pour vérifier qu'on n'avait pas de conflits.

Henault committed:
Update index.md

Henault committed:
Update index.md

Henault committed:
Modification

Henault committed:
Modification

Henault committed:
Test ajout d'un blanc

Henault committed:
Test ajout d'un blanc

Henault committed:
Update index.md

Henault committed:
Update index.md

Henault committed:
Create Media

Henault committed:
Create Media

Henault committed:
Modif Structure plus redactionnel

Henault committed:
Modif Structure plus redactionnel

Henault committed:
Merge branch 'netex-nouveaux-modes' of https://github.com/etalab/transport-profil-netex-fr into netex-nouveaux-modes

Henault committed:
Structuration du profil

Reorganisation du document et insertion d'image. Travail sur la partie generique

Henault committed:
Structuration du profil

Reorganisation du document et insertion d'image. Travail sur la partie generique

Henault committed:
Commit Atelier 20 03

Pour echange lors de l'atelier du jour

Henault committed:
Commit Atelier 20 03

Pour echange lors de l'atelier du jour

Henault committed:
Profil NeTEx FR NM v0.0

Sans Figure ni Exemples xml
=> Dans 2 fichiers à traiter par ailleurs

Henault committed:
Profil NeTEx FR NM v0.0

Sans Figure ni Exemples xml
=> Dans 2 fichiers à traiter par ailleurs

prhod commented:
Changement de la cible de la branche vers `next` après un rebase sur origin/next qui n'avait pas de conflit.
@Henault il faudra que tu mettes à jour ta branche local : 
`git reset origin/netex-nouveaux-modes` (n'hésite pas à me dire si tu as des soucis)

thbar committed:
Merge branch 'next' into netex-nouveaux-modes

thbar commented:
@Henault @TuThoThai j'ai mis à jour à nouveau la branche, qui commençait à être en retard, pour nous assurer que ca ne diverge pas.

Henault committed:
Merge branch 'netex-nouveaux-modes' of https://github.com/etalab/transport-profil-netex-fr into netex-nouveaux-modes

TuThoThai commented:
Je décale toute la partie nouveaux modes à la v2.5, ce qui nous permettra de mieux articuler cette partie avec les autres.

thbar committed:
Merge branch 'next' into netex-nouveaux-modes

Henault committed:
Merge branch 'netex-nouveaux-modes' of https://github.com/etalab/transport-profil-netex-fr into netex-nouveaux-modes

Henault committed:
Rajout Covoiturage

TuThoThai committed:
Merge branch 'v2.5-wip' into netex-nouveaux-modes

Henault committed:
Evolutions suite Atelier de Janv 2026

Henault committed:
Merge branch 'netex-nouveaux-modes' of https://github.com/etalab/transport-profil-netex-fr into netex-nouveaux-modes

Henault committed:
1eres images

Merci Pascal R

Henault committed:
Integration des images (Diag Conceptuel)

Henault committed:
Mise à jour Definition

Henault committed:
Complements de defintions

Henault committed:
Specification nb place Pooling & Sharing

Ajout d'exemple pour préciser comment spécifier le nb de places dédiées au covoiturage ou au partage de vehicules dans un parking

Henault committed:
Ajout Compte CLient
